new code

```diff
@@ -0,0 +1,163 @@
+import java.util.*;
+public class tictactoenew
+{
+    public static void main(String args[])
+    { 
+        String rerun = "yes";
+
+        while(rerun.equalsIgnoreCase("yes"))
+        {
+            System.out.print("welcome!");
+
+            int i,j,k=1,l,m;
+            String player1,player2;
```

Comment:
Give some spaces between variables
`String player1, player2;`
same for int above